concepticondata/conceptlists/Pagel-2018-200.tsv

```diff
@@ -0,0 +1,201 @@
+ID	NUMBER	ENGLISH	CONCEPTICON_ID	CONCEPTICON_GLOSS	RATE
+Pagel-2018-200-1	1	all	98	ALL	0.000185864
+Pagel-2018-200-2	2	and	1577	AND	0.000170797
+Pagel-2018-200-3	3	animal	619	ANIMAL	0.00019532
+Pagel-2018-200-4	4	ashes	646	ASH	0.00022033
+Pagel-2018-200-5	5	at	1461	AT	0.000316527
+Pagel-2018-200-6	6	back			0.000279398
+Pagel-2018-200-7	7	bad	1292	BAD	0.000430415
+Pagel-2018-200-8	8	bark			0.000266899
+Pagel-2018-200-9	9	because	1157	BECAUSE	0.000417718
+Pagel-2018-200-10	10	belly	1251	BELLY	0.000312297
+Pagel-2018-200-11	11	big	1202	BIG	0.000269116
+Pagel-2018-200-12	12	bird	937	BIRD	0.000177633
+Pagel-2018-200-13	13	bite	1403	BITE	0.000344925
+Pagel-2018-200-14	14	black	163	BLACK	0.000144405
+Pagel-2018-200-15	15	blood	946	BLOOD	0.00014066
+Pagel-2018-200-16	16	blow			0.000240553
+Pagel-2018-200-17	17	bone	1394	BONE	0.000110854
+Pagel-2018-200-18	18	breathe	1407	BREATHE	0.00014767
+Pagel-2018-200-19	19	burn	2102	BURN	0.00015978
+Pagel-2018-200-20	20	child	2099	CHILD	0.000351751
+Pagel-2018-200-21	21	cloud	1489	CLOUD	0.000190384
+Pagel-2018-200-22	22	cold	1287	COLD	0.000192004
+Pagel-2018-200-23	23	come	1446	COME	0.000157998
+Pagel-2018-200-24	24	count			0.000210452
+Pagel-2018-200-25	25	cut	1432	CUT	0.000247422
+Pagel-2018-200-26	26	day			7.73722E-05
```

Comment:
day is opposite to night (obvious from original mapping in IELex and ABVD)

Reply:
Makes sense